internal/token.go

```diff
@@ -1,9 +1,20 @@
 // Package internal contains utilities internal to the weasel packages.
 package internal
 
-import "golang.org/x/oauth2/google"
+import (
+	"golang.org/x/oauth2/google"
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2"
```

Comment:
Doesn't seem like imports are formatted. Should this go above ""golang.org/x/oauth2/google"?
I think this change might need a "go fmt".

Reply:
fixed